pkg/retry/retry.go

```diff
@@ -0,0 +1,13 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package retry
+
+import "time"
+
+const (
```

Comment:
Do we want to make this user-configurable?

Reply:
Yes, but not right now. This ties into another issue we have about making user facing per call retries. We need a formal design for this first.